src/plugins/orangehrmDashboardPlugin/Service/ChartService.php

```diff
@@ -73,39 +73,28 @@ static function (SubunitCountPair $x, SubunitCountPair $y) {
         );
     }
 
-    /**
-     * @param array $locationEmployeeCounts
-     * @return int
-     */
-    public function getLocationUnassignedEmployeeCount(array $locationEmployeeCounts): int
-    {
-        $totalActiveEmployee = $this->getChartDao()->getTotalActiveEmployeeCount();
-
-        $assignedEmployeeCount = 0;
-        foreach ($locationEmployeeCounts as $locationEmployeeCount) {
-            $assignedEmployeeCount += $locationEmployeeCount->getEmployeeCount();
-        }
-        return $totalActiveEmployee - $assignedEmployeeCount;
-    }
-
     /**
      * @param int $limit
      * @return EmployeeDistributionByLocation
      */
     public function getEmployeeDistributionByLocation(int $limit=8): EmployeeDistributionByLocation
     {
         $locationEmployeeCount = $this->getChartDao()->getEmployeeDistributionByLocation();
-        $unassignedEmployeeCount =  $this->getLocationUnassignedEmployeeCount($locationEmployeeCount);
 
-        $otherArray = [];
-        if (count($locationEmployeeCount) > $limit) {
-            $otherArray = array_slice($locationEmployeeCount, $limit);
-            $locationEmployeeCount = array_slice($locationEmployeeCount, 0, $limit);
+        $otherCount = 0;
+        $assignedEmployeeCount = 0;
+        foreach ($locationEmployeeCount as $key => $employeeCount) {
```

Comment:
```suggestion
        foreach ($locationEmployeeCount as $index => $employeeCount) {
```
refactor variable name